### PR TITLE
fix config constructors

### PR DESCRIPTION
**Pull request by p-avital**

### CODE REVIEW

File: cfg.json5 [hunk 1]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (JEnoch) [inline cfg.json5:1]: Is this file required ?

**Review by JEnoch — CHANGES_REQUESTED** [review]

File: examples/z_pub.py [hunk 2]
```diff
@@ -55,6 +55,7 @@
 
 args = parser.parse_args()
 conf = zenoh.Config.from_file(args.config) if args.config is not None else zenoh.Config()
+print(conf)
```
Comment (Mallets) [inline examples/z_pub.py:58]: The conf shouldn't be printed in the example.

**Review by Mallets — CHANGES_REQUESTED** [review]

**Review by JEnoch — APPROVED** [review]
